apps/web/src/components/Editor.tsx

```diff
@@ -13,6 +14,11 @@ import { Command } from "../lib/tiptap/command/command-extension";
 import { commandSuggestions } from "../lib/tiptap/command/command-suggestions";
 import { getCommandExtensions } from "../lib/tiptap/tiptap";
 import { Toggle } from "./ui/toggle";
+import css from "highlight.js/lib/languages/css";
+import js from "highlight.js/lib/languages/javascript";
+import ts from "highlight.js/lib/languages/typescript";
+import html from "highlight.js/lib/languages/xml";
```

Comment:
Does `highlight.js` support any other languages?
Would be nice to be able to embed Rust, Haskell, C etc. and especially Solidity given our crypto niche.

Reply:
Yeah, a bunch (including the ones you mentioned) except Solidity it seems like. And most of them should already be available without the `highlight.register` functionality: https://www.npmjs.com/package/lowlight#types